source/administration/upgrade.rst

```diff
@@ -1,241 +1,110 @@
-   - ``https://releases.mattermost.com/0.6.0/mattermost-team-0.6.0-linux-amd64.tar.gz``
-   - SHA-256 Checksum: ``9eb364f7f963af32d4a9efe3bbb5abb2a21ca5d1a213b50ca461dab047a123b6``
+Upgrading Mattermost Server
+===========================
+
+In most cases you can upgrade Mattermost Server in a few minutes, but the upgrade can take longer depending on several factors, including the size and complexity of your installation, and the version that you're upgrading from.
+
+Upgrading to the Latest Version
+-------------------------------
+
+If you are upgrading from version 3.0 or later, these instructions apply to you. If you are upgrading from a version earlier than 3.0.0, you must first `upgrade to version 3.0.3 <../administration/upgrading-to-3.0.html>`_.
+
+.. _before-you-begin:
+
+**Before you begin**
+
+Read these instructions carefully from start to finish. Make sure that you understand each step before starting the upgrade. If you have questions or concerns, you can ask on the Mattermost forum at https://forum.mattermost.org/.
+
+.. important::
+  Review the :doc:`important-upgrade-notes` to make sure you are aware of any actions you need to take before or after upgrading from your particular version.
+
+You should gather the following information before starting the upgrade:
+
+Existing install directory - *{install-path}*
+  If you don't know where Mattermost Server is installed, use the ``whereis platform`` command. The output should be similar to */opt/mattermost/bin/platform*. The install directory is everything before the last occurrence of the string */mattermost*. In this example, the *{install-path}* is ``/opt``.
+Location of your local storage directory
+  The local storage directory contains all the files that users have attached to their messages. If you don't know its location, open the System Console and go to **Files > Storage** and read the value in **Local Storage Directory**. Relative paths are relative to the ``mattermost`` directory. For example, if the local storage directory is ``./data/`` then the absolute path is ``{install-path}/mattermost/data``.
+Owner and group of the install directory - *{owner}* and *{group}*
+  Use the ``ls -l {install-path}/mattermost/bin/platform`` command to get the owner and group.
+
+**To upgrade Mattermost Server**:
+
+.. note::
+  If you are upgrading an HA cluster, `review these upgrade notes instead <https://docs.mattermost.com/deployment/cluster.html#upgrade-guide>`_.
+
+1. Review the :doc:`important-upgrade-notes` to make sure you are aware of any actions you need to take before or after upgrading from your particular version.
+
+2. In a terminal window on the server that hosts Mattermost Server, change to your home directory.
+
+  ``cd ~``
+
+3. Delete any files and directories that might still exist from a previous download.
+
+  .. code-block:: text
+
+    rm mattermost*.gz
+    rm -r mattermost

```

Comment:
@coreyhulen Yeah, I submitted an update to the upgrade guide last month but Jeff mentioned the correct steps are correct. Wondering if you have thoughts on that? Happy to make changes if you see fit.

PR: https://github.com/mattermost/docs/pull/1367

Jeff's note:

```
Step 3 deletes any previously downloaded files in the home directory. It doesn't delete any files in the running Mattermost instance. In step 6, the config.json is copied from the running Mattermost instance to the home directory.

The Mattermost server is stopped in Step 7 instead of Step 1 to help ensure that it is stopped for the shortest time possible during the upgrade.

If you follow the existing procedure on your own instance, copy/pasting from the guide to your terminal window, you'll see that it works perfectly as-is. I think this PR is trying to fix something that isn't broken.
```